Lib/fontTools/merge.py

```diff
@@ -436,6 +442,67 @@ class CmapUnicodePlatEncodings:
 	BMP = {(4, 3, 1), (4, 0, 3), (4, 0, 4), (4, 0, 6)}
 	FullRepertoire = {(12, 3, 10), (12, 0, 4), (12, 0, 6)}
 
+def _is_Default_Ignorable(u):
```

Comment:
for those as ignorant as me a sentence clarifying what "ignorable" means in this context would be most appreciated :D

Reply:
Would be nice if this was able to drive itself off unicode data and thus updated by just updating the data files (or the version of something like unicodedata2). Maybe not worthwhile for this alone but it's fairly easy to imagine subsets of unicode datafiles ending up scattered all over the place.

Reply:
> for those as ignorant as me a sentence clarifying what "ignorable" means in this context would be most appreciated :D

"Default_Ignorable" is a Unicode technical term. I'll add a URL.

Reply:
> Would be nice if this was able to drive itself off unicode data and thus updated by just updating the data files (or the version of something like unicodedata2). Maybe not worthwhile for this alone but it's fairly easy to imagine subsets of unicode datafiles ending up scattered all over the place.

Yes. We have this general problem in many places. For much of our unicode data we derive it using scripts. But those of this scale end up being manual.

Reply:
we could parse DerivedCoreProperties.txt in MetaTools/buildUCD.py, like we do for blocks or scripts and make them available in fontTools.unicodedata, maybe in a separate PR